frontend/packages/knative-plugin/src/utils/create-knative-utils.ts

```diff
@@ -28,12 +28,13 @@ interface ResourceType {
 
 export const createKnativeService = (
   name: string,
+  applicationName: string,
   namespace: string,
   scaling: ServerlessScaling,
   limits: LimitsData,
   { targetPort },
-  imageStreamName: string,
-  imageStreamTag?: string,
+  imageStreamUrl: string,
+  imageStreamName?: string,
```

Comment:
if we can pass `imageStreamName` in `deploy-submit-utils` no need to have it as optional param